docs/General/Manuals/client-manual.rst

```diff
@@ -1691,7 +1692,7 @@ and Wonders.
 
 Overall happiness of the citizens in your cities depends heavily on all of these factors. Certain forms of
 government have varying degrees of empire size penalties and as your empire grows you will have to deal with
-the negative consequences of managing a large empire and the unhappiness it creates. The amount of Luxury
+the negative consequences of managing a large empire and the unhappiness it creates. The amount of luxury
 Goods you are producing as part of your `Tax Rates View`_ will aid this problem. Certain city improvements can
```

Comment:
This is now "luxury Goods", should it be all lowercase?

Reply:
Gold, Tax(es) and Luxury Goods are specific game elements and also very prominent in the GUI and I think should be capitalized.